client/src/components/home/Home.tsx

```diff
@@ -0,0 +1,141 @@
+
+export interface HomeProps {
+  companyName: string;
+  theme?: PartialTheme | Theme;
+}
+
+const menuProps: IContextualMenuProps = {
+  items: [
+    {
+      key: 'host',
+      text: 'as host (presenter)'
+    },
+    {
+      key: 'attendee',
+      text: 'as attendee'
+    }
+  ]
+  // By default, the menu will be focused when it opens. Uncomment the next line to prevent this.
+  // shouldFocusOnMount: false
+};
+
+export const HomeComponent = (props: HomeProps): JSX.Element => {
+  return (
+    <Stack styles={fullScreenStyles}>
+      <Stack
+        horizontalAlign="center"
+        verticalAlign="start"
+        styles={containerStyles(props.theme)}
+        tokens={{ childrenGap: 15 }}
+      >
+        <Stack styles={innerContainer}>
+          <Stack verticalAlign="center" horizontalAlign="center">
+            <Image width="23.75rem" height="6.25rem" src={imageCalendar} alt="calendarImage"></Image>
+          </Stack>
+          <Stack>
+            <Text styles={lineHeight28px}>Hello,</Text>
+            <Text styles={lineHeight22px}>What would you like to do?</Text>
+            <Stack horizontal horizontalAlign="space-between" styles={btnStackStyles}>
+              <StackItem>
+                <DefaultButton text="Book an appointment" styles={buttonStyle} iconProps={{ iconName: 'Calendar' }} />
+              </StackItem>
+              <StackItem>
+                <DefaultButton
+                  iconProps={{ iconName: 'Video' }}
+                  text="Start a call"
+                  split
+                  splitButtonAriaLabel="See 2 options"
+                  aria-roledescription="split button"
+                  menuProps={menuProps}
+                  styles={buttonStyle}
+                />
+              </StackItem>
+              <StackItem>
+                <DefaultButton text="Join from link" styles={buttonStyle} iconProps={{ iconName: 'Link' }} />
+              </StackItem>
+            </Stack>
+            <Text styles={font16pxStyle}>Frequently asked questions</Text>
+            <Stack styles={btnStackStyles}>
+              <Link tabIndex={0} data-testid="bookingsSetupLink" target="_blank">
+                <Stack horizontal disableShrink>
+                  <StackItem align="center" style={linkStyles}>
+                    How do I change my Microsoft Bookings page URL?
+                  </StackItem>
+                  <StackItem align="center" style={textDecorationNone}>
+                    <span style={newWindowIconWrapper}>
+                      <Icon iconName="OpenInNewWindow" />
+                    </span>
+                  </StackItem>
+                </Stack>
+              </Link>
+              <Link tabIndex={0} data-testid="bookingsSetupLink" target="_blank">
+                <Stack horizontal disableShrink>
+                  <StackItem align="center" style={linkStyles}>
+                    How do I switch from using Rooms (preview) to using Microsoft Bookings?
```

Comment:
Let's leave this out for now since this feature is not yet complete and we don't yet have a doc to link this to.

Reply:
done